docker/servicesmw/Jenkinsfile

```diff
@@ -0,0 +1,195 @@
+// todo use it when border settings for handling `-k8s` suffix will be prepared
+// def sandbox = params.sandbox.endsWith("-k8s") ?  params.sandbox : params.sandbox + '-k8s';
+
+def kubectlImage = "artifactory.wikia-inc.com/ops/k8s-deployer:0.0.15"
+def mediawikiImageName = "mediawiki-servicesmw"
+def mediawikiImage = "artifactory.wikia-inc.com/sus/${mediawikiImageName}"
```

Comment:
why `sus`?

Reply:
it seems that almost everything is stored in /sus/ : https://artifactory.wikia-inc.com/artifactory/webapp/#/search/package/eyJwYWNrYWdlUGF5bG9hZCI6W3siaWQiOiJkb2NrZXJWMkltYWdlIiwidmFsdWVzIjpbIiprOHMtZGVwbG95ZXIqIl19XSwic2VsZWN0ZWRQYWNrYWdlVHlwZSI6eyJpZCI6ImRvY2tlclYyIiwiaWNvbiI6ImRvY2tlciIsImRpc3BsYXlOYW1lIjoiRG9ja2VyIn19